paths/release_triggers/create.yaml

```
@@ -0,0 +1,91 @@
+tags:
+  - Release Triggers
+parameters:
+  - "$ref": "../../parameters.yaml#/X-PhraseApp-OTP"
+  - "$ref": "../../parameters.yaml#/account_id"
+  - "$ref": "../../parameters.yaml#/distribution_id"
+requestBody:
+  required: true
+  content:
+    application/json:
+      schema:
+        type: object
+        title: release/create/parameters
+        properties:
+          cron_schedule:
+            description: Cron schedule for the scheduler. Read more about the format of this field at https://en.wikipedia.org/wiki/Cron
+            type: string
+            example: "15 18 1,3 * *"
+          time_zone:
+            description: Time zone for the scheduler
+            type: string
+            example: "Europe/Berlin"
+          locale_ids:
+            description: List of locale ids that will be included in the release.
+            type: array
+            items:
+              type: string
+            example:
+              - abcd1234cdef1234abcd1234cdef1234
+              - fff565db236400772368235db2c6117e
+          tags:
+            description: Only include tagged keys in the release. For a key to be included it must be tagged with all tags provided
+            type: array
+            items:
+              type: string
+            example:
+              - android
+              - feature1
+          branch:
+            description: Branch used for release
+            type: string
+            example: my-feature-branch
+          app_min_version:
+            description: The created releases will be available only for apps with version greater or equal to this value
+            type: string
+            example: "1.0.0"
+          app_max_version:
+            description: The created releases will be available only for apps with version less or equal to this value
+            type: string
+            example: "2.0.0"
+responses:
+  "201":
+    description: Created
+    content:
+      application/json:
+        schema:
+          "$ref": "../../schemas/release_trigger.yaml#/release_trigger"
+    headers:
+      X-Rate-Limit-Limit:
+        "$ref": "../../headers.yaml#/X-Rate-Limit-Limit"
+      X-Rate-Limit-Remaining:
+        "$ref": "../../headers.yaml#/X-Rate-Limit-Remaining"
+      X-Rate-Limit-Reset:
+        "$ref": "../../headers.yaml#/X-Rate-Limit-Reset"
+  "400":
+    "$ref": "../../responses.yaml#/400"
+  "404":
+    "$ref": "../../responses.yaml#/404"
+  "429":
+    "$ref": "../../responses.yaml#/429"
+x-code-samples:
+  - lang: Curl
+    source: |-
+      curl "https://api.phrase.com/v2/accounts/:account_id/distributions/:distribution_id/release_triggers" \
+        -u USERNAME_OR_ACCESS_TOKEN \
+        -X POST \
+        -d '{"cron_schedule":"15 18 1,3 * *","time_zone":"Europe/Berlin","locale_ids":["abcd1234cdef1234abcd1234cdef1234","fff565db236400772368235db2c6117e"],"tags":["android","feature1"],"branch":"my-feature-branch","app_min_version":"1.0.0","app_max_version":"2.0.0"}' \
```

same as above if we want to change the cron example to align more with our use cases:

```suggestion
        -d '{"cron_schedule":"15 18 * * 0,1","time_zone":"Europe/Berlin","locale_ids":["abcd1234cdef1234abcd1234cdef1234","fff565db236400772368235db2c6117e"],"tags":["android","feature1"],"branch":"my-feature-branch","app_min_version":"1.0.0","app_max_version":"2.0.0"}' \
```